src/Controls/src/Core/RadioButtonGroupController.cs

```diff
@@ -28,17 +28,17 @@ public RadioButtonGroupController(Compatibility.Layout<View> layout)
 				UpdateGroupNames(layout, _groupName);
 			}
 
-			WeakReferenceMessenger.Default.Register<RadioButtonGroupController, RadioButtonGroupSelectionChanged>(this, HandleRadioButtonGroupSelectionChanged);
-			WeakReferenceMessenger.Default.Register<RadioButtonGroupController, RadioButtonGroupNameChanged>(this, HandleRadioButtonGroupNameChanged);
-			WeakReferenceMessenger.Default.Register<RadioButtonGroupController, RadioButtonValueChanged>(this, HandleRadioButtonValueChanged);
+			WeakReferenceMessenger.Default.Register<RadioButtonGroupController, RadioButtonGroupSelectionChanged>(this, (r,m) => r.HandleRadioButtonGroupSelectionChanged(m));
+			WeakReferenceMessenger.Default.Register<RadioButtonGroupController, RadioButtonGroupNameChanged>(this, (r,m) => HandleRadioButtonGroupNameChanged(m));
```

Comment:
Missed `r.` here? 🤔
I also would recommend making the lambdas `static` to make the code more explicit and less error prone 🙂
```suggestion
			WeakReferenceMessenger.Default.Register<RadioButtonGroupController, RadioButtonGroupNameChanged>(this, static (r,m) => r.HandleRadioButtonGroupNameChanged(m));
```

Reply:
> I also would recommend making the lambdas static

Ah, added in C# 9. TIL.